docs/Example HOWTO.md

```diff
@@ -233,90 +238,316 @@ dns2.example.com. 3600    IN      A       192.168.0.20
+```
+; Nameserver records    Class  Type     Target
+dns1.example.com.         IN     A      192.168.0.18
+dns2.example.com.         IN     A      192.168.0.20
+rds1.example.com.         IN     A      192.168.0.50
+rds2.example.com.         IN     A      192.168.0.51
+```
 
-Lastly we provide the IP addresses for the hosts in the system. This file can of course be expanded to contain names for all the hosts, end-points, and switches in the system, making debugging simpler:
+### Checking the DNS records for Multiple RDSs
 
 ```
-; Nameserver records
-dns1.example.com.    IN      A       192.168.0.18
-dns2.example.com.    IN      A       192.168.0.20
-rds1.example.com.    IN      A       192.168.0.50
-rds2.example.com.    IN      A       192.168.0.51
+# dig _nmos-register._tcp.example.com PTR
+ <<>> DiG 9.11.4-P2-RedHat-9.11.4-26.P2.amzn2.5.2 <<>> _nmos-register._tcp.example.com PTR
+;; global options: +cmd
+;; Got answer:
+;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 48397
+;; flags: qr aa rd ra; QUERY: 1, ANSWER: 2, AUTHORITY: 2, ADDITIONAL: 3
+
+;; OPT PSEUDOSECTION:
+; EDNS: version: 0, flags:; udp: 4096
+;; QUESTION SECTION:
+;_nmos-register._tcp.example.com. IN	PTR
+
+;; ANSWER SECTION:
+_nmos-register._tcp.example.com. 3600 IN PTR	reg-api-2._nmos-register._tcp.example.com.
+_nmos-register._tcp.example.com. 3600 IN PTR	reg-api-1._nmos-register._tcp.example.com.
+
+;; AUTHORITY SECTION:
+example.com.		3600	IN	NS	dns2.example.com.
+example.com.		3600	IN	NS	dns1.example.com.
+
+;; ADDITIONAL SECTION:
+dns1.example.com.	3600	IN	A	192.168.0.18
+dns2.example.com.	3600	IN	A	192.168.0.20
+
+;; Query time: 0 msec
+;; SERVER: 10.0.50.59#53(10.0.50.59)
+;; WHEN: Tue Sep 06 07:29:22 UTC 2022
+;; MSG SIZE  rcvd: 178
+```
+
+As can be seen the DNS server provides two PTRs to the query.
+Details on prioritization and TXT can be checked as well:
+
+```
+# dig reg-api-2._nmos-register._tcp.example.com SRV
+
+; <<>> DiG 9.11.4-P2-RedHat-9.11.4-26.P2.amzn2.5.2 <<>> reg-api-2._nmos-register._tcp.example.com SRV
+;; global options: +cmd
+;; Got answer:
+;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 5862
+;; flags: qr aa rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 2, ADDITIONAL: 4
+
+;; OPT PSEUDOSECTION:
+; EDNS: version: 0, flags:; udp: 4096
+;; QUESTION SECTION:
+;reg-api-2._nmos-register._tcp.example.com. IN SRV
+
+;; ANSWER SECTION:
+reg-api-2._nmos-register._tcp.example.com. 3600	IN SRV 20 10 80 rds2.example.com.
+
+;; AUTHORITY SECTION:
+example.com.		3600	IN	NS	dns1.example.com.
+example.com.		3600	IN	NS	dns2.example.com.
+
+;; ADDITIONAL SECTION:
+rds2.example.com.	3600	IN	A	192.168.0.51
+dns1.example.com.	3600	IN	A	192.168.0.18
+dns2.example.com.	3600	IN	A	192.168.0.20
+
+;; Query time: 0 msec
+;; SERVER: 10.0.50.59#53(10.0.50.59)
+;; WHEN: Tue Sep 06 07:32:09 UTC 2022
+;; MSG SIZE  rcvd: 192
+```
+
```

Comment:
```suggestion

And now the TXT record for the same service:

```
# dig reg-api-2._nmos-register._tcp.example.com TXT

; <<>> DiG 9.11.4-P2-RedHat-9.11.4-26.P2.amzn2.5.2 <<>> reg-api-2._nmos-register._tcp.example.com TXT
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 41920
;; flags: qr aa rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 2, ADDITIONAL: 3

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
;; QUESTION SECTION:
;reg-api-2._nmos-register._tcp.example.com. IN TXT

;; ANSWER SECTION:
reg-api-2._nmos-register._tcp.example.com. 3600	IN TXT "api_ver=v1.0,v1.1,v1.2,v1.3" "api_proto=http" "pri=20" "api_auth=false"

;; AUTHORITY SECTION:
example.com.		3600	IN	NS	dns2.example.com.
example.com.		3600	IN	NS	dns1.example.com.

;; ADDITIONAL SECTION:
dns1.example.com.	3600	IN	A	192.168.0.18
dns2.example.com.	3600	IN	A	192.168.0.20

;; Query time: 0 msec
;; SERVER: 10.0.50.59#53(10.0.50.59)
;; WHEN: Tue Sep 13 09:59:12 UTC 2022
;; MSG SIZE  rcvd: 217

```

```